python-package/mlbox/encoding/categorical_encoder.py

```diff
@@ -9,7 +9,7 @@
 import os
 
 # Set the keras backend if not set, default is theano
-if not "KERAS_BACKEND" in os.environ:
+if "KERAS_BACKEND" not in os.environ:
     os.environ["KERAS_BACKEND"] = "theano"
 
 from keras.layers.core import Dense, Reshape, Dropout
```

Comment:
In my opinion there is no reason to define KERAS_BACKEND in the module. This variable is already manage by keras with the file $HOME/.keras/keras.json created during the installation of keras. But if there is a really need to so, I think you can define it in the categorical/__init__.py file